backend/mem/file.go

```diff
@@ -0,0 +1,219 @@
+package mem
+
+import (
+	"bytes"
+	"errors"
+	"github.com/c2fo/vfs/v4"
+	"io"
+	"time"
+)
+
+//File implements vfs.File interface for os fs.
+type File struct {
+	exists 		bool
+	timeStamp 	time.Time
+	isOpen		bool
+	isZB		bool
```

Comment:
This might be dumb, but what is "isZB"?

Reply:
isRef and privSlice could probably use end of line comments describing their meaning/purpose as well...